docs/plugins/testing_workshop_docs/Testing-Resources.md

```diff
@@ -0,0 +1,17 @@
+# Testing resources
+
+These resources may be helpful in developing tests.  
+[Today’s plugin](https://github.com/DragaDoncila/plugin-tests) - Plugin Tests
```

Comment:
I think we've lost the bullet points here at some stage so those need to come back. You can make a bullet list by just adding a `-` at the beginning of each item.

Reply:
I will add back after we pull this down.